src/components/Toast/__stories__/toast.stories.scss

```diff
@@ -35,3 +36,9 @@
     width: 390px;
   }
 }
+
+// overriding margin for storybook wrapper
+.monday-storybook_multiple-story-elements-wrapper .monday-style-toast{
+  margin-right:auto;
+  margin-left:auto;
```

Comment:
What is this about? What have u fixed?

Reply:
the initial toast had transform: transitionX(-50%) , i remove it and used margin auto,but there is a legacy selector that Hadas wrote only in story book that had Heighr specificity. 
This solves this.